spec/controllers/api/v0/orders_controller_spec.rb

```diff
@@ -175,6 +175,14 @@ module Api
               .map{ |o| o[:id] }).to eq serialized_orders([order2, order3, order1, order4, order7])
                 .map{ |o| o["id"] }
           end
+
+          it 'can sort orders by billing_address.lastname' do
+            get :index, params: { q: { s: 'billing_address_lastname ASC' } },
```

Comment:
I can't find this parameter in the codebase, only `billing_address_name`. Shall we use a real example? Interestingly the scope for ransack, `sort_by_billing_address_name_asc`, is doing a `join` and therefore ignoring records without address. I'm a bit lost in this mess. Do you have time to look into this, @dacook?

Reply:
Honestly I'm a bit lost too, I just came in to review this little chunk of code.. 
I'm hoping that @Matt-Yorkley will have prior knowledge, and be able to to say what's what?

Reply:
I feel confident that I could work this out but I would need to spend a bit of time on it. Maybe tomorrow unless Matt looked at this already.